addons/binding/org.openhab.binding.netatmo/META-INF/MANIFEST.MF

```diff
@@ -8,13 +8,14 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ClassPath: .,
  lib/retrofit-1.9.0.jar,
  lib/okhttp-2.3.0.jar,
+ lib/joda-time-2.9.9.jar,
```

Comment:
We have Java 8, please use the Java 8 dates instead of joda time

Reply:
Yes, I am not running the swagger-codegen command with the right parameters as it keeps requiring the joda-time library. Anyone knows what the correct set of parameters is usually used on the netatmo swagger declaration file?

Reply:
Fixed

Reply:
It still seems to be in this manifest file?

Reply:
Sorry. Missed it from the commit

Reply:
Would you please post the swagger-codegen command you used? Including all parameters? That would be some kind of documentation for others who want to improve this binding in the future. Thank you very much.